docs/pages/desktop-access/getting-started.mdx

```diff
@@ -299,14 +299,15 @@ windows_desktop_service:
     #
     # For example, if your domain is "example.com", the NetBIOS name for it is
     # likely "EXAMPLE". When connecting as the "svc-teleport" user, you should
-    # use the format: "EXAMPLE\svc-teleport".
+    # use the format: 'EXAMPLE\svc-teleport'.
     #
     # If you are unsure of your NetBIOS name, you can find it by opening a PowerShell command prompt
     # and running:
-    # ```
+    # 
     # (Get-ADDomain).NetBIOSName
-    # ```
-    username: "$LDAP_USERNAME"
+    # 
+    # Note the use of single quotes due to the \ character
```

Comment:
Since there is no `\` character here, I think this comment makes more sense up above where it says "use the format"

Reply:
moved